app/views/admin/events/_form.html.haml

```diff
@@ -0,0 +1,77 @@
+.row
+  .col-md-12
+    .page-header
+      %h1
+        New Event
+.row
+  .col-md-12
+    = semantic_form_for(@event, url: @url) do |f|
+      = f.inputs name: 'Event Information' do
```

Comment:
I'll think on how to make it more DRY,
another thing - should be have submitter/speaker fields at the very bottom, these are not 2nd grade fields in this context, that's why I put them at the beginning of the form